### ISSUE TITLE
Some league urls gives blank results

### ISSUE THREAD

**Issue by spilliton** [tsbicca/espn_fantasy_football_alternate_history#1]
Love this site btw :+1: 

One of my leagues works and the other does not, here is the non working one:
http://games.espn.go.com/ffl/clubhouse?leagueId=293172

Takes me to http://toiletbowlchamp.com/league/293172 and shows:

![screen shot 2015-11-19 at 9 36 05 am](https://cloud.githubusercontent.com/assets/131613/11274964/02390f26-8ea1-11e5-9ce1-4222fbd0b615.png)

**Comment by kvaratskhelia** [comment]
Check to see if the leagues that don't work are viewable to the public. The setting is the last setting in basic settings on the league settings page.

**Comment by spilliton** [comment]
Yep that looks like that is is.  Maybe a more helpful message could be shown in this instance?
